docs/tasks.rst

```diff
@@ -1755,6 +1760,11 @@ Options
 
 	 Parameter to pass to the Apex. Use as %%%PARAM_2%%% in the Apex code.Defaults to an empty value.
 
+``--minimize_apex MINIMIZEAPEX``
+	 *Optional*
+
+	 Removes block comments, inline comments, and whitespace at the start of lines.  Default: False
+
```

Comment:
@davisagli , this is the only change I want.

I called make docs, and all of the other changes appeared.   Any suggestions how to proceed?

Reply:
@spelak-salesforce sorry, this is because I made a change to the generated task doc format but forgot to actually regenerate it. You may keep these changes or remove them.

Long term we are going to stop storing this file in the repository, and only generate it while building the docs.

Reply:
I'll keep the changes then if they're ok to merge.